### DISCUSSION TITLE
Ctrl Z and Ctrl Y

### DISCUSSION THREAD

**Discussion by RustoMCSpit** [labithiotis/zed-themes#102] [General]
undo and redo commands

**Comment by labithiotis** [comment]
undo and redo have been implemented, let me know how it goes :)

**Reply by RustoMCSpit** [reply]
no icons to click for people using phones

**Reply by labithiotis** [reply]
Although it "works" on mobile I don't expect anyone to use phones to edit a theme. Although, it guess it can be argued using a tablet is possible and more likely. I feel UI is already cluttered enough that adding another two icons is not ideal, we could consider adding icons to only show on small breakpoint but tbh I think the mobile/sm UI needs it own solution and at this stage its not a priority.

**Reply by RustoMCSpit** [reply]
any solution?

**Reply by labithiotis** [reply]
Added undo/redo buttons that only show for devices without keyboards i.e mobile/tablets.

![Screen Shot 2025-05-22 at 08 37 00](https://github.com/user-attachments/assets/9e434721-f150-4941-a225-303b6037d5c4)
